lib/activegraphql/query.rb

```diff
@@ -0,0 +1,80 @@
+require 'httparty'
+require 'active_support/inflector'
+require 'activegraphql/support/fancy'
+
+module ActiveGraphQL
+  class Query < Support::Fancy
```

Comment:
Yes, I guess a bit late for the first release :smile:.
Anyway, the basic idea about the mapper looks good, even I don't understand the relation with `ActiveRecord` here.
So, I encourage you to make coding proposal to have a deeper discussion on that.

Thanks!